src/visual.ts

```diff
@@ -580,11 +642,17 @@ module powerbi.extensibility.visual {
                     color: settings.labels.fill
                 };
 
+                if (nodes.filter( (node: SankeyDiagramNode) => {
```

Comment:
Small code style comment.
There shouldn't be a white space after the first round bracket